README.md

```diff
@@ -99,6 +116,16 @@ Refering $SPARK_HOME to the Spark installation directory. This library has been
 | kinesis.client.numRetries |     3 |  Maximum Number of retries for Kinesis API requests  |
 | kinesis.client.retryIntervalMs |     1000 |  Cool-off period before retrying Kinesis API  |
 
+## Kinesis Sink Configuration
+ Option-Name        | Default-Value           | Description  |
+| ------------- |:-------------:| -----:|
+| sink.streamName   | - | Name of the stream in Kinesis to write to|
+| sink.endpointUrl  | https://kinesis.us-east-1.amazonaws.com |  The aws endpoint of the kinesis Stream |
+| awsAccessKeyId |    -     |    AWS Credentials for  Kinesis describe, read record operations    
+| awsSecretKey |      -  |    AWS Credentials for  Kinesis describe, read record |
+| kinesis.recordMaxBufferedTime | 1000 (millis) | Specify the maximum buffered time of a record |
```

Comment:
nit: Shall we make it  `kinesis.executor.recordMaxBufferedTime` or `kinesis.producer.recordMaxBufferedTime`
Similar comment for ` kinesis.maxConnections`